src/main/java/net/cytonic/cytosis/messaging/RabbitMQ.java

```diff
@@ -42,32 +44,93 @@ public void initializeQueues() {
         try {
             channel.queueDeclare(SERVER_DECLARE_QUEUE, false, false, false, null);
         } catch (IOException e) {
-            Logger.error("An error occoured whilst initializing the 'SERVER_DECLARE_QUEUE'.", e);
+            Logger.error("An error occurred whilst initializing the 'SERVER_DECLARE_QUEUE'.", e);
         }
         try {
             channel.queueDeclare(SHUTDOWN_QUEUE, false, false, false, null);
         } catch (IOException e) {
-            Logger.error("An error occoured whilst initializing the 'SHUTDOWN_QUEUE'.", e);
+            Logger.error("An error occurred whilst initializing the 'SHUTDOWN_QUEUE'.", e);
+        }
+        try {
+            channel.exchangeDeclare("chat-testing", BuiltinExchangeType.FANOUT);
+            channel.queueDeclare(CHAT_CHANNEL_QUEUE, false, false, false, null);
+            channel.queueBind(CHAT_CHANNEL_QUEUE,"chat-testing","");
```

Comment:
I think the chanel name (the "chat-testing"), not sure if its actually a channel, but it should probably be a variable